apps/todolist-mongo-go/Dockerfile

```diff
@@ -1,18 +1,18 @@
 #FROM registry.access.redhat.com/ubi8/ubi-minimal
-FROM golang:1.17.8-alpine AS build-env
+FROM golang:1.23-alpine AS build-env
 RUN mkdir /build
 WORKDIR /build
-COPY apps/todolist-mongo-go/*.go .
-COPY apps/todolist-mongo-go/go.mod .
-COPY apps/todolist-mongo-go/go.sum .
+COPY *.go .
+COPY go.mod .
+COPY go.sum .
 RUN go mod download
 RUN go mod tidy
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -a -installsuffix cgo -o app
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} go build -v -a -installsuffix cgo -o app
 
 FROM scratch
 COPY --from=build-env /build/app /app
-COPY apps/todolist-mongo-go/resources/ /resources/
-COPY apps/todolist-mongo-go/index.html .
+COPY resources/ /resources/
+COPY index.html .
```

Comment:
same here

Reply:
same answer.. working from the mig-demo-apps/apps/todolist-mongo-go
 dir not outside of it.